apps/docs/components/Navigation/NavigationMenu/TopNavBar.tsx

```diff
@@ -118,6 +87,11 @@ const TopNavBar: FC = () => {
               </a>
             </Button>
           )}
+          {process.env.NEXT_PUBLIC_DEV_AUTH_PAGE === 'true' && (
```

Comment:
I set this on Vercel Development/Preview. Add to local dev if you need to sign in locally

Reply:
you can probably create a feature flag on config cat to toggle this behaviour to only appear on local dev / staging for the team

Reply:
but does config cat know who you are?

the problem is that you're anonymous on docs staging, so how would you get authed so configcat can identify you?